.changeset/serious-buses-punch.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap/guided-answers-extension-webapp': minor
```

Comment:
Solution looks good, thank you!
However, I'd say according to https://semver.org/ this is a patch, not a minor update ;)

```suggestion
'@sap/guided-answers-extension-webapp': patch
```

Reply:
That makes sense. Thanks, Klaus